README.md

```diff
@@ -1,2 +1,20 @@
-# cruise-ships
-this is a object oriented programming module and it is based on cruise ships
+Cruise Ships 🚢
+Javascript Object-Oriented code for a Cruise Ships.
+Building and working with many objects that allow a cruise ship to operate.
+Running the app
+To run clone to local machine.
+
+Navigate to directory and install npm:
+
+npm install
+To install jest:
+
+npm install -D jest
+To run tests:
+
+npm run test
+Built Using
+Node.js
+Jest
+Author
+Joseph redmond
```

Comment:
Your readme is a really important file, its worth spending a bit of time to make sure it has the right info in it and also that it looks good too! It should contain:

- installation instructions (ie cloning the repo and starting it up)
- what technologies are required to use it, what the application dependencies are (eg `jest` and `node`)
- a description of what the app actually does, what its features are and how to use it
- how the app is tested
- what you would do if you had more time (if applicable)

Regards presentation, readmes are written in markdown language, which is a simple but quite powerful in what it can do.  I find [markdown live preview](https://markdownlivepreview.com/) is useful as it enables you to play with what the file will look like without having to do multiple commits to the repo to see the results, have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too

As a basic example, try this
```
# Cruise Ships 🚢
An exercise in Javascript Object-Oriented code. The task was to build an app that makes use of JS objects to build an application in Node that represents behaviour of cruise ships.

## Available scripts

### Installation

Clone the repository, `cd` into the new directory and run
```
npm install
```

### Running tests
```
npm test
```


## Dependencies
- Node.js
- Jest


## Author
[Joseph redmond](your-linked-in-page-url-here)
```